res/controllers/Reloop-Mixage.scripts.js

```diff
@@ -0,0 +1,780 @@
+            for (let effectUnit = 1; effectUnit <= Mixage.numEffectUnits; effectUnit++) {
+                engine.setValue(`[EffectRack1_EffectUnit${effectUnit}]`, fxChannel, false);
+            }
+        } else {
+            engine.setValue(`[EffectRack1_EffectUnit${unitNr}]`, fxChannel, true);
+        }
+    }
+};
+
+// This function is necessary to allow for soft takeover of the filter amount button
+// see https://github.com/mixxxdj/mixxx/wiki/Midi-Scripting#soft-takeover
+Mixage.handleFilter = function(_channel, _control, value, _status, group) {
+    const amountValue = value / 127;
+    Mixage.amountValue[group] = amountValue;
+    engine.setValue(`[QuickEffectRack1_${group}]`, "super1", value / 127);
+};
+
+// Handles setting soft takeovers when pressing shift
+Mixage.handleShift = function(_channel, _control, value, _status, group) {
+    const unitNr = script.deckFromGroup(group);
+    if (value === DOWN) {
+        if (engine.getValue(`[QuickEffectRack1_${group}]`, "super1") !== Mixage.amountValue[group]) {
+            engine.softTakeoverIgnoreNextValue(`[QuickEffectRack1_${group}]`, "super1");
+        }
+    } else {
+        const effectUnit = `[EffectRack1_EffectUnit${unitNr}]`;
+        const focusedEffect = engine.getValue(effectUnit, "focused_effect");
+        if (focusedEffect === 0) {
+            if (engine.getValue(effectUnit, "super1") !== Mixage.amountValue[group]) {
+                engine.softTakeoverIgnoreNextValue(effectUnit, "super1");
+            }
+        } else {
+            const effectSlot = `[EffectRack1_EffectUnit${unitNr}_Effect${focusedEffect}]`;
+            if (engine.getValue(effectSlot, "meta") !== Mixage.amountValue[group]) {
+                engine.softTakeoverIgnoreNextValue(effectSlot, "meta");
+            }
+        }
+    }
+};
+
+// The "disc" button that enables/disables scratching
+Mixage.scratchToggle = function(_channel, _control, value, _status, group) {
+    if (value === DOWN) {
+        Mixage.scratchPressed[group] = true;
+        Mixage.scratchTogglePressTimer[group] = engine.beginTimer(Mixage.shortPressTimeout, function() {
+            Mixage.scratchTogglePressTimer[group] = 0;
+        }, true);
+    } else {
+        Mixage.scratchPressed[group] = false;
+        if (Mixage.scratchTogglePressTimer[group] !== 0) {
+            engine.stopTimer(Mixage.scratchTogglePressTimer[group]);
+            Mixage.scratchTogglePressTimer[group] = 0;
+            Mixage.stopLoopAdjust(group);
+            Mixage.scratchToggleState[group] = !Mixage.scratchToggleState[group];
+            Mixage.toggleLED(Mixage.scratchToggleState[group], group, "scratch_active");
+            if (Mixage.scrollToggleState[group]) {
+                Mixage.scrollToggleState[group] = !Mixage.scrollToggleState[group];
+                Mixage.toggleLED(Mixage.scrollToggleState[group], group, "scroll_active");
+            }
+        }
+    }
+};
+
+// The "loupe" button that enables/disables track scrolling
```

Comment:
:wolf: or :loop: ; )

Reply:
https://en.wikipedia.org/wiki/Loupe
"Magnifying glass" is a tad bit on the long side. Other suggestions?

Reply:
aaah, okay, thought it's a typo and you mean loop.
In that case loupe hopefully works